src/js/view/view.js

```diff
@@ -645,6 +645,9 @@ function View(_api, _model) {
     }
 
     function _nativeFullscreenChangeHandler(event) {
+        if (!OS.mobile) {
```

Comment:
This is needed to not cause the player container to enter fullscreen when entering PiP mode. The below code looks like we should only run it for iOS (`jw-flag-ios-fullscreen`)

Reply:
nice!